React/tic-tac/src/redux/auth/actions.js

```diff
@@ -0,0 +1,32 @@
+import Service from '@services/UserService';
+
+export const actions = { LOGIN: 'LOGIN', LOGOUT: 'LOGOUT' };
+
+function logInTrue(payload) {
+  return {
+    type: actions.LOGIN,
+    loggedIn: payload
+  };
+}
+
+function logIn(payload) {
+  return dispatch => {
+    Service(payload)
+      .then(response => {
+        if (response.ok) {
+          dispatch(logInTrue(true));
```

Comment:
You can dispatch the object you're returning in you `logInTrue` function. :)

Reply:
Please change this function call (and the function itself) for the action object.